### ISSUE TITLE
InducingHomomorphism type piracy

### ISSUE THREAD

**Issue by blegat** [kalmarek/SymbolicWedderburn.jl#22]
Defining a constructor for `InducingHomomorphism` with a vector of monomial outside of this package is type piracy, I would suggest something like
```julia
(::InducingHomomorphism)(::AbstractVector{<:AbstractMonomial}, ::AbtractAcction)
```
If a package defines a subtype of `AbtractAcction`, it can define a method without type piracy.
Fixing this might be done at the same time as fixing https://github.com/kalmarek/SymbolicWedderburn.jl/issues/21

**Comment by kalmarek** [comment]
@blegat resolved by #25?

**Comment by blegat** [comment]
Yes :)
